tests/cxx/quoted_includes_first.h

```diff
@@ -0,0 +1,38 @@
+//===--- quoted_includes_first.cc - test input file for iwyu --------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
I think we prefer old-school include guards, but for test headers we typically don't bother. Please remove the `#pragma` for now -- there are portability subtleties around `#pragma once` that I would prefer to keep out of the test suite.

Reply:
Done.